src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -378,6 +379,13 @@ class KinDynComputations {
      */
     iDynTree::Transform getWorldTransform(std::string frameName);
 
+    /**
+     * Return the transforms as a homogeneous matrices where the frame is the frame
+     * specified by frameIndex, and the reference frame is the world one
```

Comment:
What is `frameIndex` in this case?

Reply:
Left over from copying something else.... changed to :

    

>  * Return the transforms as a homogeneous matrices where the frame is
>      * specified by name in the frameNames vector, and the reference frame is the world one
>      * (world_H_frame).